features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/MockResponses.js

```diff
@@ -31,9 +31,48 @@ class MockResponses {
             scopes: [
                 'string',
             ],
-            schemaTypes: [
-                'OAS2', 'OAS3', 'WSDL1', 'WSDL2', 'GRAPHQL_SDL', 'ASYNC_API',
-            ],
+        };
+        return Promise.resolve({ body: response });
+    }
+
+    /**
+    *
+    * Return API created from service
+    * @export
+    * @returns {*} Mocked Service Catalog model
+    */
+    static createApiFromService() {
+        const response = {
+            id: '87624513-0123-6274-9013-012345678901',
+            name: 'New-API',
+            context: '/newapi',
+            description: 'This is a new API created from a service',
+            version: '1.0.0',
+        };
+        return Promise.resolve({ body: response });
+    }
+
+    /**
+    *
+    * Return sample mocked model data for addSampleService
+    * @export
+    * @returns {*} Mocked Service Catalog model
+    */
+    static addSampleService() {
+        const response = {
+            id: '87624513-0123-6274-9013-012345678901',
+            name: 'Sample-Endpoint',
+            displayName: 'Sample-Endpoint',
+            description: 'A sample Catalog Entry that exposes a REST endpoint',
```

Comment:
Shall we correct this to 'A sample Catalog **entry** that exposes a REST endpoint'